src/z3c/rml/attr.py

```diff
@@ -142,20 +149,18 @@ def fromUnicode(self, value):
             except ValueError:
                 pass
         raise ValueError(
-            '"%s" is not a valid value. %s' %(
-                value, getFileInfo(self.context)))
+            '"%s" is not a valid value. %s' % (
+                value, getFileInfo(self.context)
+            )
+        )
 
 
 class Text(RMLAttribute, zope.schema.Text):
     """A simple unicode string."""
 
 
-if six.PY2:
-    class String(RMLAttribute, zope.schema.Bytes):
-        """A simple Bytes string."""
-else:
-    class String(Text):
-        """Formerly a simple Bytes string, now the same as Text."""
+class String(Text):
+    """Formerly a simple Bytes string, now the same as Text."""
```

Comment:
I would drop the class altogether. That much of a reason to make this a 4.0 release.

Reply:
done